pages/DSNP/Types/Profile.md

```diff
@@ -8,7 +8,7 @@ The reference content *MUST be of profile type*.
 | Field | Description | Serialization | Parquet Type | Bloom Filter |
```

Comment:
I only just noticed that `Profile` doesn't have a `Data Type` column. I think this was just an oversight. I'm ok with fixing in this or a different PR.